Reply/app/src/main/java/com/materialstudies/reply/ui/MainActivity.kt

```diff
@@ -179,10 +179,24 @@ class MainActivity : AppCompatActivity(),
     private fun setBottomAppBarForCompose() {
         binding.run {
             bottomAppBar.performHide()
-            fab.hide()
-            // Hide the BottomAppBar to avoid it showing above the keyboard
-            // when composing a new email.
-            bottomAppBar.visibility = View.GONE
+            // Get a handle on the animator that hides the bottom app bar so we can wait to hide
+            // the fab and bottom app bar until after it's exit animation finishes.
+            bottomAppBar.animate().setListener(object : Animator.AnimatorListener {
```

Comment:
Extend `AnimatorListenerAdapter` so don't need to implement un-used methods.

Reply:
Done!